components/Rating/Rating.css

```diff
@@ -0,0 +1,5 @@
+@import url(../../style.css);
+
+.rating {
+  color: #fad400;
```

Comment:
may be you can use color variable here

Reply:
Thanks for the review Neha! Here the value has been used instead of a variable because I will be using this color only in this one place hence it doesn't need to be in a variable